index.js

```diff
@@ -25,6 +57,55 @@ function Changelog( options ) {
 	}
 }
 
+Changelog.ticketParsers = {
+	// Order of keys is important!
+	// When passed several ticketTypes, they will be re-ordered in the order of the keys of this map
+	// This works around github's (or other) regex to be greedy
```

Comment:
Order is irrelevant here. I'm also not sure what's greedy about the GitHub regex; it should never match a JIRA ticket.